ro_help/hub/templates/mail/ngo_registration_notice.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+{% language 'ro' %}
+Bună, <br><br>
+
+ONG-ul tău a fost înregistrat în platforma <a href="https://rohelp.ro">RoHelp</a> şi poate fi găsit <a href="{{base_path}}{% url 'ngo-detail' ngo_id %}">aici</a>. 
```

Comment:
This probably has to be marked for translation, ideally, with `{% blocktrans %}`

Reply:
I'm not exactly sure what the better solution is in this case. I used ``language`` only to make sure ``url`` fills in the URL for the page written in Romanian, since the template is written in this language. Other templates don't use the ``url`` tag which I was going to fix in a separate PR :D 
With regards to ``blocktrans``, it might be a bit tricky to figure out the language preference of the email recipient. I didn't have a chance to look deeper into how Django chooses the preferred language, but from my container, I'd always get the site in English. That was the rationale behind using ``language`` in the first place.
I think a separate ticket should be made regarding the language issues, because they're pretty mixed and inconsistent.
Let me know what you think.

Reply:
Probably something to address in a different PR, but, a short answer to how does Django know in which language to translate this template:

`_email_ngo_owners_about_creation` has access to a `request` object from which Django can determine what language to use. 

I think that we shouldn't force `ro` language in template because this will lead to having this template never being translated. :D

Reply:
Yeah, but that request comes from the action initiated by the **admin user**, NOT the _**end recipient of the e-mail**_. Basically, to determine the language the e-mail should be in, we'd have to track the language for each user that we send the e-mail to.
Alternatively, it can be decided that all e-mails will be in Romanian (for example) so we won't need to wrap it in ``blocktrans``.